packages/client/src/container-modules.ts

```diff
@@ -42,12 +42,19 @@ import modelHintsModule from './features/hints/di.config';
 import glspHoverModule from './features/hover/di.config';
 import layoutModule from './features/layout/di.config';
 import navigationModule from './features/navigation/di.config';
+import glspRoutingModule from './features/routing/di.config';
 import glspSelectModule from './features/select/di.config';
 import sourceModelWatcherModule from './features/source-model-watcher/di.config';
 import svgMetadataModule from './features/svg-metadata/di.config';
 import toolFeedbackModule from './features/tool-feedback/di.config';
 import toolPaletteModule from './features/tool-palette/di.config';
-import { enableDefaultToolsOnFocusLossModule, toolsModule } from './features/tools/di.config';
+import changeBoundsToolModule from './features/tools/change-bounds/di.config';
+import deletionToolModule from './features/tools/deletion/di.config';
+import edgeCreationToolModule from './features/tools/edge-creation/di.config';
+import edgeEditToolModule from './features/tools/edge-edit/di.config';
+import { enableDefaultToolsOnFocusLossModule } from './features/tools/enable-default-tools-on-focus-loss';
```

Comment:
Unrelated, but maybe you could fix it while you are at it:
Looks like the `enable-default-tools-on-focus-loss` module is not default exported (in contrast to all other modules)

Reply:
Ah yes, I'm always happy to provide more consistency!